src/for-developers/wallet-faucet.md

```diff
@@ -0,0 +1,22 @@
+# Fuel Wallet
+
+You can deploy a contract without signing with a wallet by running the following command:
+
+`forc deploy --unsigned`
+
+You can initialize a wallet and an account for your wallet to sign transactions
+
+To initialize a wallet you can use `forc wallet init`. It will ask you to choose a password to encrypt your wallet. After the initialization is done you will have your mnemonic phrase.
+
+After you have an initialized wallet, you can create an account for it by `running forc wallet new`. It will ask your password to decrypt the wallet before creating an account.
```

Comment:
If would be nice to actually use `console` code blocks to show the commands and their outputs here.

Reply:
decided to remove this section entirely and send readers to the official steps, so that we have one source of truth, but noted!